### ISSUE TITLE
doesn't work with v2 of the patches posted upstream

### ISSUE THREAD

**Issue by chutz** [DHowett/framework-laptop-kmod#13]
On my Framework 13 Ryzen 7840, when trying to load cros with a 6.8.5 kernel drivers with the v2 patcheset, I get this in my dmesg:
```
$ dmesg | grep cros
[    2.365684] cros_ec_lpcs cros_ec_lpcs.0: loaded with quirks 00000002
[    2.366295] cros_ec_lpcs cros_ec_lpcs.0: couldn't reserve region1
[    2.366848] cros_ec_lpcs: probe of cros_ec_lpcs.0 failed with error -16
[    4.031967] framework_laptop framework_laptop: framework_laptop: failed to find EC cros-ec-dev.
```
Going back to the v1 patchest makes things work again.
```
[    3.976497] cros_ec_lpcs cros_ec_lpcs.0: loaded with quirks 00000003
[    3.986984] cros_ec_lpcs cros_ec_lpcs.0: Chrome EC device registered
```

**Comment by Steve-Tech** [comment]
Hi, I have a feeling you've somehow applied the v3 4/4 patch, instead of the v2 4/4 one when applying the rest of the v2 patches. Because one of the quirks were removed in the v3 (and fixed in the 3.05 BIOS), and it seems only that remaining quirk is being loaded for you, from the `loaded with quirks 00000002` vs `loaded with quirks 00000003`.

**Comment by DHowett** [comment]
Since v3 landed, I'm closing this one out - it's been verified by a couple other kernel contributors. Thanks for the debugging!
